localstack/services/cloudformation/cloudformation_starter.py

```diff
@@ -889,7 +909,19 @@ def cloudformation_backend_describe_change_set(self, change_set_name, stack_name
 
     def cloudformation_backend_execute_change_set(self, change_set_name, stack_name=None):
         change_set_name = change_set_name.replace(TEST_AWS_ACCOUNT_ID, MOTO_CFN_ACCOUNT_ID)
-        return cloudformation_backend_execute_change_set_orig(self, change_set_name, stack_name)
+        resp = cloudformation_backend_execute_change_set_orig(self, change_set_name, stack_name)
+
+        stack = self.change_sets.get(change_set_name, None)
+        if not stack:
+            for cs in self.change_sets:
+                if self.change_sets[cs].change_set_name == change_set_name:
+                    stack = self.change_sets[cs]
+
+        stack.output_map = stack._create_output_map()
```

Comment:
To avoid overwriting existing output maps (also in case there are upstream changes), should we change this to the following:
```
stack.output_map = stack.output_map or stack._create_output_map()
```

Reply:
`stack.output_map` is always existing, It was created in `stack.initialize_resources()`
`    def initialize_resources(self):
        self.resource_map.load()
        self.resource_map.create(self.template_dict)
        self.output_map.create()
        self._add_stack_event("CREATE_COMPLETE")
        self.status = "CREATE_COMPLETE"`